server/buttonAlerts.js

```diff
@@ -41,15 +41,15 @@ async function handleValidRequest(button, numButtonPresses) {
     const currentTime = await db.getCurrentTime(pgClient)
 
     if (currentSession === null || currentTime - currentSession.updatedAt >= helpers.getEnvVar('SESSION_RESET_TIMEOUT')) {
-      currentSession = await db.createSession(button.id, CHATBOT_STATE.STARTED, numButtonPresses, null, null, null, pgClient)
+      currentSession = await db.createSession(button.id, CHATBOT_STATE.STARTED, numberOfAlerts, null, null, null, pgClient)
     } else {
-      currentSession.incrementButtonPresses(numButtonPresses)
+      currentSession.incrementButtonPresses(numberOfAlerts)
```

Comment:
Here you could either hardcode the value 1 or, if you're willing to make another change in `brave-alert-lib`, remove the parameter entirely and always increment by one.

Reply:
I'll make a small update to brave-alert-lib. :)